.github/actions/get-deploy-paths/action.yml

```
@@ -30,23 +35,50 @@ outputs:
   spack-config:
     description: Path to the ACCESS-NRI/spack-config repository associated with the install of spack.
     value: ${{ steps.path.outputs.spack-config }}
+  spack-packages-root:
+    description: |
+      Path to the folder containing all ACCESS-NRI/spack-packages repositories used in the install of spack.
+      For Release inputs.deployment-environment, this is the repository itself.
+      For Prerelease inputs.deployment-environment, this is the folder containing multiple spack-packages repositories.
+    value: ${{ steps.path.outputs.spack-packages-root }}
   spack-packages:
-    description: Path to the ACCESS-NRI/spack-packages repository associated with the install of spack.
+    description: Path to the ACCESS-NRI/spack-packages repository associated with the environment created in the install of spack.
     value: ${{ steps.path.outputs.spack-packages }}
 runs:
   using: composite
   steps:
-    - name: Get ${{ inputs.deployment-environment }} Remote Paths
-      id: path
+    - name: Validate Inputs
+      id: valid
       shell: bash
+      env:
+        ACCEPTABLE_DEPLOYMENT_ENVIRONMENT_TARGETS: Release Prerelease
       run: |
+        # Check that the inputs.spack-installs-root-path exists - it's usually a var
         if [ -z "${{ inputs.spack-installs-root-path }}" ]; then
```

```suggestion
        if [ -z '${{ inputs.spack-installs-root-path }}' ]; then
```
Minor, but for context directly coming from inputs, I would use single quotes to avoid any unexpected behaviour in case the input contains characters that bash would expand (like `!` or `$`).